dae/dae/annotation/effect_annotator.py

```diff
@@ -215,13 +225,17 @@ def _do_annotate(
             length=length
         )
 
+        gene_list = AnnotationEffect.gene_effects(effects)[1]
+        LGD_gene_list = AnnotationEffect.lgd_gene_effects(effects)
         # r = AnnotationEffect.wrap_effects(effects)
         full_desc = AnnotationEffect.effects_description(effects)
         result = {
             "worst_effect": full_desc[0],
             "gene_effects": full_desc[1],
             "effect_details": full_desc[2],
             "allele_effects": AlleleEffects.from_effects(effects),
+            "gene_list": gene_list,
+            "LGD_gene_list": LGD_gene_list
```

Comment:
consider adding tests for effect annotator gene_list and LGD_gene_list attributes